pom.xml

```diff
@@ -124,6 +124,7 @@
         <jaeger.version>1.8.1</jaeger.version>
         <opentracing.version>0.33.0</opentracing.version>
         <opentracing-kafka.version>0.1.13</opentracing-kafka.version>
+        <opentelemetry.alpha-version>1.18.0-alpha</opentelemetry.alpha-version>
```

Comment:
So let me explain the gist of this better ...
Right now in the OpenTelemetry land, some libraries have a "definitive" version, while others are still in "alpha" because evolving quickly. 
For bridge support I preferred to use `opentelemetry.version` property to refer the "definitive" version, while `opentelemetry.alpha-version` for the "alpha" ones that I would expect to disappear at some point.
For example, I am using `opentelemetry.version` property in the 3rd library pom files where the `opentelemetry-exporter-jaeger` dependency is a final release but at same time I have `opentelemetry.alpha-version` used for the `opentelemetry-kafka-clients-2.6` and `opentelemetry-sdk-extension-autoconfigure` which are still marked as "alpha". The same rule applies here.

Reply:
Seems a bit weird if you have only the `alpha` version. Maybe it should be at least called `opentelemetry-alpha.version`? Anyway, I can live with this, so feel free to keep it.